README.md

```diff
@@ -0,0 +1,65 @@
+# Uyuni RFCs
+
+Many changes, including bug fixes and documentation improvements, can be implemented and reviewed via the normal GitHub pull request workflow. Some changes though are "substantial" - those should be put through an appropriate design process and produce a consensus among the Uyuni core team.
+
+The "RFC" (request for comments) process is intended to provide a consistent and controlled path for new features to enter the project. This process will be adjusted over time as more features are implemented and the community settles on specific approaches to feature development.
```

Comment:
```suggestion
The "RFC" (request for comments) process is intended to provide a consistent and controlled path for such substantial changes to enter the project. This process will be adjusted over time as more features are implemented and the community settles on specific approaches to feature development.
```

Reply:
I would agree, considering the RFC history, but OTOH, how do we create incentives for the community to keep future documents updated?